cmd/plugin/util/util.go

```diff
@@ -47,16 +47,16 @@ func PrintError(e error) {
 }
 
 // ParseVersionString returns the major, minor, and patch numbers of a version string
-func ParseVersionString(v string) (int, int, int, error) {
+func ParseVersionString(v string) (major, minor, patch int, err error) {
 	parts := versionRegex.FindStringSubmatch(v)
 
 	if len(parts) != 4 {
 		return 0, 0, 0, fmt.Errorf("could not parse %v as a version string (like 0.20.3)", v)
 	}
 
-	major, _ := strconv.Atoi(parts[1])
-	minor, _ := strconv.Atoi(parts[2])
-	patch, _ := strconv.Atoi(parts[3])
+	major, _ = strconv.Atoi(parts[1])
```

Comment:
hum why is this error being ignored? (please don't change this PR, I'm just asking why golangci-lint is ignoring it!)

Also, as a followup, we should probably get all the semver parsers on our code and start using some semver library :)

Reply:
Good catch! I reran the golangci-lint just now, the linter reported about 50 errcheck errors, let me try fixing those.

Reply:
Fixed the errors